contentcuration/contentcuration/static/js/edit_channel/views.js

```diff
@@ -723,10 +723,10 @@ var BaseWorkspaceListNodeItemView = BaseListNodeItemView.extend({
 			'add_nodes', 'add_topic');
 	},
 	make_droppable:function(){
-		if(this.model.get("kind") === "topic"){
-			var DragHelper = require("edit_channel/utils/drag_drop");
-			DragHelper.addTopicDragDrop(this, this.open_folder, this.handle_drop);
-		}
+		// if(this.model.get("kind") === "topic"){
+		// 	var DragHelper = require("edit_channel/utils/drag_drop");
+		// 	DragHelper.addTopicDragDrop(this, this.open_folder, this.handle_drop);
+		// }
```

Comment:
If we're going to leave commented-out code in the repo, it would be good to leave a comment indicating why it was removed and linking to the relevant trello card